dev/tests/integration/testsuite/Magento/GraphQl/_files/product_downloadable_with_purchased_separately_links_rollback.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
Please add `declare(strict_types=1);` to each new classes

Reply:
**done**

Reply:
**done**